### DISCUSSION TITLE
here's a way to unblock any website

### DISCUSSION THREAD

**Discussion by LieutenantGst** [3kh0/ext-remover#1095] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
How to unblock any site in a few minutes

1. Make a Google doc paper (docs.new)
2. Add some links like Spotify or Youtube
3. click any of the links
4. Its going to work trust me works with securly!

please don't exit the website you used because if you try again it'll block it.

**Comment by AshtonDavies** [comment]
Securly is mad crazy if it ignores blocked sites redirected from Google Docs.

**Reply by CGRmags** [reply]
FR wth if this is real its crazy

**Reply by LieutenantGst** [reply]
@AshtonDavies yeah securly is crazy for letting us on blocked websites through Google docs without them being blocked. @CGRmags i saw a exploit like this on a GitHub page so I tired it then it didn’t work then I found this way that I said and it worked it should work if you have securly on your chromebook

**Reply by AshtonDavies** [reply]
This will probably get patched not too long in the future just like the translate method.

**Reply by S-PScripts** [reply]
> This will probably get patched not too long in the future just like the translate method.

this exploit has existed for like 5 months yk
this works on other blockers too

**Reply by S-PScripts** [reply]
however for other blockers you need to save the document as a pdf first then open the links from the pdf file
